sidechain/storage/src/storage.rs

```diff
@@ -254,53 +313,61 @@ impl<SignedBlock: SignedBlockT> SidechainStorage<SignedBlock> {
 		}
 	}
 
-	/// adds the block to the WriteBatch
+	/// Adds the block to the WriteBatch.
 	fn add_last_block(&mut self, batch: &mut WriteBatch, block: &SignedBlock) {
 		let hash = block.hash();
 		let block_number = block.block().block_number();
 		let shard = block.block().shard_id();
-		// Block hash -> Signed Block
+		// Block hash -> Signed Block.
 		SidechainDB::add_to_batch(batch, hash, block);
 
-		// (Shard, Block number) -> Blockhash (for block pruning)
+		// (Shard, Block number) -> Blockhash (for block pruning).
 		SidechainDB::add_to_batch(batch, (shard, block_number), hash);
 
-		// (last_block_key, shard) -> (Blockhash, BlockNr) current blockchain state
+		// (last_block_key, shard) -> (Blockhash, BlockNr) current blockchain state.
 		let last_block = LastSidechainBlock { hash, number: block_number };
 		self.last_blocks.insert(shard, last_block); // add in memory
 		SidechainDB::add_to_batch(batch, (LAST_BLOCK_KEY, shard), last_block);
 	}
 
-	/// delete block to the WriteBach
+	/// Add delete block to the WriteBatch.
 	fn delete_block(
 		&self,
 		batch: &mut WriteBatch,
 		block_hash: &H256,
 		block_number: &BlockNumber,
 		shard: &ShardIdentifierFor<SignedBlock>,
 	) {
-		// Block hash -> Signed Block
+		// Block hash -> Signed Block.
 		SidechainDB::delete_to_batch(batch, block_hash);
-		// (Shard, Block number) -> Blockhash (for block pruning)
+		// (Shard, Block number) -> Blockhash (for block pruning).
 		SidechainDB::delete_to_batch(batch, (shard, block_number));
 	}
 
-	/// delete last block & add to the last block (write batch only)
+	/// Add delete last block to WriteBatch and remove block from memory.
```

Comment:
`Add delete` this is quite confusing. In fact, I don't undertand. :)

Reply:
Hm. Would this be better?
`/// Add delete command to remove last block to WriteBatch and remove it from memory.`

Reply:
I still find it unclear. The header of a function doc should be concise and should not care about internals. Hence, I'd go for:

`/// Delete the last block from the DB.`

Reply:
Maybe some quotation marks can help in the comment here? -> `Add 'delete last block' command to WriteBatch [..]`

Reply:
Ok, now I at least understand, what this function does. 😄 

I think you can't entail everything this method does, but we should think of a naming scheme that clearly discerns, which methods write to the db and which not. This you can only see from the docs now, which is the wrong place IMO.

But this should not be solved now. For now, I can agree to: 
```
 /// Add delete command to remove last block to WriteBatch and remove it from memory.
```